src/components/MonthCalendar.tsx

```diff
@@ -77,78 +75,101 @@ const MonthCalendar = ({ month, events, year }: MonthCalendarProps): JSX.Element
     };
 
     return (
-        <div style={{
-            display: 'flex',
-            height: '100%',
-        }}>
-            <div
-                style={{
-                    marginRight: '0.5rem',
-                    position: 'relative',
-                    width: '1rem',
-                }}>
+        <>
+            <View position="absolute" right="15rem" top="-2.6rem">
+                <Flex alignItems="center">
+                    <ActionButton data-testid="back-button" onPress={
+                        () => onFocusDate(new Date(new Date(focusDate).setDate(focusDate.getDate() - 30)))
+                    }>
+                        <Msg id="misc.calendar.prev" />
+                    </ActionButton>
+                    <View data-testid="selected-month" padding="size-100">
+                        <FormattedDate
+                            month="long"
+                            value={ new Date(year, month + 1, 0) }
+                            year="numeric"
+                        />
+                    </View>
+                    <ActionButton data-testid="fwd-button" onPress={
+                        () => onFocusDate(new Date(new Date(focusDate).setDate(focusDate.getDate() + 30)))
```

Comment:
I think it should be possible to shift by exactly one month (instead of 30 days) by doing this:

```ts
new Date(focusDate.getFullYear(), focusDate.getMonth() + 1, focusDate.getDate());
```

Reply:
wow, didn't know this would work, but it does. thought i'd have to do extra calculations to make it work across years.